packages/agent-utils/src/agent/ChatMessageBuilder.ts

```diff
@@ -23,7 +23,7 @@ export class ChatMessageBuilder {
     return {
       role: "function",
       name: funcName,
-      content: result || "undefined"
+      content: result
```

Comment:
Can content be undefined?
From the OpenAI SDK: `The contents of the message. `content` is required for all messages except assistant messages with function calls.`
I think this means that content is always required for role "function"

Reply:
sorry i meant to also update the constructor so result is not `undefined` - just committed the change

Reply:
nice